R/aedseo.R

```diff
@@ -101,6 +107,13 @@ aedseo <- function(
   res <- tibble::tibble()
   skipped_window <- base::rep(FALSE, base::nrow(tsd))
 
+  # Add the seasons to tsd if available
+  if (!is.null(season)) {
+    tsd <- tsd |> dplyr::mutate(Season = epi_calendar(time))
+  } else {
+    tsd <- tsd |> dplyr::mutate(Season = "Not defined")
+  }
```

Comment:
These mutates create a column with sentence case, but the other columns are snake case. I suggest keeping everything as snake case.